File: C/palindrome_check.c
// Check the word that whether is a palindrome or not //

#include <stdio.h>
#include <string.h>
#define SIZE 20

int main(void)
{
    char string[SIZE+1];
    char new_str[SIZE+1];
   
    gets(string);
    
    int length = strlen(string);
    int index = length-1;
    int x = 0;
    while(index >= 0){
        new_str[x] = string[index];
        x++;
        index--;
    }

    if(strcmp(new_str, string) == 0){
        printf("Palindrome");
    }
    else{
        printf("Not Palindrom");
    }

    

    return 0;
}

File: Python/palindrome_check.py
# Palindrome check by Python #

string = input()
new_str = []

length = len(string)
index = length - 1
while index >= 0:
    new_str.append(string[index])
    index-=1

i = 0
while i < length:
    if(new_str[i] == string[i]):
        if(i == length-1):
            print("The initial string:", string)
            print("The reverse string: ", *new_str, sep="")
            print("This string is the Palindrome")
            break
        i+=1
    else:
        print("The initial string:", string)
        print("The reverse string: ", *new_str, sep="")
        print("This string is not the Palindrome")
        break